app/stac_api/validators_serializer.py

```diff
@@ -57,152 +57,6 @@ def validate_json_payload(serializer):
         raise serializers.ValidationError(code='payload', detail=errors)
 
 
-def validate_asset_href_path(item, asset_name, path):
```

Comment:
I noticed that these weren't called anywhere in the code (except from the tests). I guess uncalled code can be removed then?!

Reply:
Surprising. But yes, if not used anywhere move it to the tests. 
File name suggests it is(was?) used in serializers at some point, but just to be sure: guess you checked the admin as well right?
Just one thing: currently we derive the asset path from collection_name/item_name/asset_filename. With meteoschweiz we'll have the case that we'll have to accept asset paths that are not on our platform. So this validation function might come handy then. Maybe it's an idea to mark it for removal but leave it for the moment. Or we re/move it now and add checks again once we know what exactly we have to check

Reply:
No it didn't occur to me to check the admin. Also it could be that it somehow is used implicitly with getattr or something. Will do further tests!

Reply:
Maybe @ltshb knows more. He seems to have made this.

Reply:
So as I see it, the usage of this function was removed with this PR: https://github.com/geoadmin/service-stac/pull/194